### DISCUSSION TITLE
TabView navContent pass through is not working as intended

### DISCUSSION THREAD

**Discussion by kemgn** [primefaces/community#883] [PrimeReact] [answered]
I have a react component like this. I'm trying to use TabView. 

**Here is its code:** 
```
export default function ModuleMain() {
  return (
    <TabView
      pt={{
        root: {
          className: 'prime-tab-root',
        },
        navContent: {
          className: 'prime-tab-header',
        },
        nav: {
          className: 'prime-tab-header',
        }
      }}>
      <TabPanel header="Views">
        <Views />
      </TabPanel>
      <TabPanel header="Properties">
        <Properties />
      </TabPanel>
    </TabView>
  )
}

```

**And here is my CSS code:** 
`prime-tab-header{
    border-radius: 12px;
}

.prime-tab-root{
    height: calc(100% - 56px);
}`

"nav" and "root" pt is working perfectly fine but navContent is not working as I intended. CSS class doesnt apply to it. Anyone can see where I'm doing wrong?

**Comment by melloware** [comment, marked as answer]
Looks like its fixed in PR 10.3.1: https://stackblitz.com/edit/vpbgmg?file=src%2FApp.jsx
